locales/en.json

```diff
@@ -18,10 +21,26 @@
 		"description": "Please select the mode that you would like to use",		
 		"title": "Welcome to Velocity!"
 	},
+	"notifications": {
+		"clearAll": "clear all notifications",
+		"new": "new",
+		"noNew": "no new notifications"
+	},
+	"pageNames": {
+		"about": "About",
+		"chatRoom": "Chat Room",
+		"grades": "Grades",
+		"logout": "Logout",
+		"profile": "Profile",
+		"projects": "Projects",
+		"settings": "Settings",
```

Comment:
I think we should clarify this to 'App Settings' as well as internal references, as Settings imply generally all but we only offer app settings within. Worse, profs will also have to contest with this, and a separate 'Profile Settings'